src/core/js/scriptLoader.js

```diff
@@ -30,7 +30,8 @@
                 a11y: 'libraries/jquery.a11y',
                 scrollTo: 'libraries/scrollTo',
                 bowser: 'libraries/bowser',
-                'enum': 'libraries/enum'
+                enum: 'libraries/enum',
+                jqueryMobile: 'libraries/jquery.mobile.custom'
```

Comment:
i've added an issue to touch-only loading > https://github.com/adaptlearning/adapt_framework/issues/1648 as this has merged already. i think we've hit a majority consensus on doing that?